kairon/shared/data/data_models.py

```
@@ -0,0 +1,1341 @@
+    email: constr(to_lower=True, strip_whitespace=True)
+    role: ACCESS_ROLES = ACCESS_ROLES.TESTER.value
+    activity_status: ACTIVITY_STATUS = ACTIVITY_STATUS.INACTIVE.value
+
+    @validator("email")
+    def validate_email(cls, v, values, **kwargs):
+        from kairon.shared.utils import Utility
+
+        try:
+            Utility.verify_email(v)
+        except AppException as e:
+            raise ValueError(str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)